Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -887,8 +887,13 @@ private bool DoHandleActionGetAndWieldItem(WorldObject item, Container itemRootO
             return true;
         }
 
+        public static bool UseWieldRequirements = true;
```

Comment:
This can't be static, it will break if work is done on multiple threads.

Reply:
Follow up question, where is this set to false?

Reply:
this is an internal debugging only var

Reply:
ahh, ok. A comment on it would be good. I saw it before during the refactor and was confused.